### PR TITLE
PIP-796-add-bsmooth

### PR DESCRIPTION
**Pull request by paul-sud**
The formatting is funny to my eyes due to being accustomed to black-formatted Python, but I just autoformatted it in R Studio.

### CODE REVIEW

**Comment by paul-sud** [comment]
Ok found an R autoformatter to add to pre-commit, much less hacky than using RStudio.

**Comment by paul-sud** [comment]
I don't seem to have permissions to invite @meenakshikagda to the pipeline-devs team (preferable to adding her as  a collaborator for just this repo), but I would like to get her review on the wgbs_pipeline/bsmooth.R script as well.

File: Dockerfile [hunk 1]
```diff
@@ -1,45 +1,80 @@
...
+    cd gemBS && git checkout 6d7a8ab25c2c44e6c6cca1485bba5b5fbaafc88f && \
+    python3 setup.py install --user
 ENV PATH="/root/.local/bin:${PATH}"
 
-COPY wgbs_pipeline wgbs_pipeline
-ENV PATH="/software/wgbs_pipeline:${PATH}"
+# Instal UCSC v377 bedToBigBed util
+RUN git clone https://github.com/ENCODE-DCC/kentUtils_bin_v377 && \
+    rm $(find kentUtils_bin_v377/bin/ -type f -not -path '*bedToBigBed')
+ENV PATH=${PATH}:/software/kentUtils_bin_v377/bin
```
Comment (ottojolanki) [inline Dockerfile]:
```suggestion
ENV PATH="${PATH}:/software/kentUtils_bin_v377/bin"
```

File: Dockerfile [hunk 2]
```diff
@@ -1,45 +1,80 @@
...
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/software/kentUtils_bin_v377/lib
```
Comment (ottojolanki) [inline Dockerfile]:
```suggestion
ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/software/kentUtils_bin_v377/lib"
```

File: Dockerfile (hunk 1 above)
Comment (ottojolanki) [reply Dockerfile]: to be consistent.

**Review by ottojolanki — COMMENTED** [review]
LGTM with couple nits to Dockerfile

**Pull request merged**